src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Color = 'red'| 'green' | 'blue';
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
```

Comment:
I think we can make do this with union literal type
type Shape = 'triangle' | 'circle' | 'rectangle'

Reply:
it would be better to use enum